src/lib/pubkey/pkcs8.h

```diff
@@ -15,8 +15,10 @@
 #include <chrono>
 #include <functional>
 #include <memory>
+#include <optional>
 #include <span>
 #include <string_view>
+#include <variant>
```

Comment:
Seems orphaned and unused. I'm guessing you considered it to distinguish duration and iterations.